src/package/nuspec/Microsoft.TestPlatform.Portable.nuspec

```diff
@@ -0,0 +1,466 @@
+﻿<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Microsoft.TestPlatform.Portable</id>
+    <version>15.0.0</version>
+    <title>Microsoft.TestPlatform.Portable</title>
```

Comment:
How do we validate functionality of this new package?

Reply:
@codito Manually verified P0 scenarios. 
If the Acceptance Test case be part of next PR, please approve and merge this PR. 
Else I will add it.

Reply:
Done.